addons/hr_timesheet/report/report_timesheet_templates.xml

```diff
@@ -72,21 +72,25 @@
                         <div class="oe_structure"/>
                         <div class="row mt8">
                             <div class="col-12">
-                                <t t-if="doc.allow_timesheets and doc.timesheet_ids">
-                                    <h1 class="my-4">
-                                        <t t-if="not show_record">
-                                            <t t-out="title"/>: <span t-field="doc.name"/>
-                                        </t>
-                                    </h1>
-                                    <h2>
-                                        <span>Timesheets
-                                            <t t-if="show_record">
-                                                for the <t t-out="doc.name"/> <t t-out="title"/>
+                                <t t-set="tasks" t-value="doc.timesheet_ids and not show_task and doc.child_ids and doc + doc.child_ids or doc"/>
```

Comment:
This is very hard to read, I would suggest rewriting this using ternary operators